nistats/first_level_model.py

```diff
@@ -0,0 +1,561 @@
+        This parameter specifies the hemodynamic response function (HRF) for
+        the design matrices. It can be 'canonical', 'canonical with derivative'
+        or 'fir'.
+
+    drift_model : string, optional
+        This parameter specifies the desired drift model for the design
+        matrices. It can be 'polynomial', 'cosine' or 'blank'.
+
+    period_cut : float, optional
+        This parameter specifies the cut period of the low-pass filter in
+        seconds for the design matrices.
+
+    drift_order : int, optional
+        This parameter specifices the order of the drift model (in case it is
+        polynomial) for the design matrices.
+
+    fir_delays : array of shape(n_onsets) or list, optional
+        In case of FIR design, yields the array of delays used in the FIR
+        model.
+
+    min_onset : float, optional
+        This parameter specifies the minimal onset relative to the design
+        (in seconds). Events that start before (slice_time_ref * t_r +
+        min_onset) are not considered.
+
+    mask: Niimg-like, NiftiMasker or MultiNiftiMasker object, optional,
+        Mask to be used on data. If an instance of masker is passed,
+        then its mask will be used. If no mask is given,
+        it will be computed automatically by a MultiNiftiMasker with default
+        parameters.
+
+    target_affine: 3x3 or 4x4 matrix, optional
+        This parameter is passed to nilearn.image.resample_img. Please see the
+        related documentation for details.
+
+    target_shape: 3-tuple of integers, optional
+        This parameter is passed to nilearn.image.resample_img. Please see the
+        related documentation for details.
+
+    low_pass: False or float, optional
+        This parameter is passed to nilearn.signal.clean.
+        Please see the related documentation for details.
+
+    high_pass: False or float, optional
+        This parameter is passed to nilearn.signal.clean.
+        Please see the related documentation for details.
+
+    smoothing_fwhm: float, optional
+        If smoothing_fwhm is not None, it gives the size in millimeters of the
+        spatial smoothing to apply to the signal.
+
+    memory: string, optional
+        Path to the directory used to cache the masking process and the glm
+        fit. By default, no caching is done. Creates instance of joblib.Memory.
+
+    memory_level: integer, optional
+        Rough estimator of the amount of memory used by caching. Higher value
+        means more memory for caching.
+
+    standardize : boolean, optional
+        If standardize is True, the time-series are centered and normed:
+        their variance is put to 1 in the time dimension.
+
+    signal_scaling: bool, optional,
+        If True, fMRI signals are scaled to the mean value of scaling_axis
+        Incompatible with standardize (standardize=False is enforced when\
+        signal_scaling is True).
+
+    scaling_axis: int or tuple, optional,
```

Comment:
signal_scaling and scaling_axis should become one single parameter, that would be one of {False, 0, 1, (0,1)}